### PR TITLE
chore: fix og image issue

### PR DESCRIPTION
**Pull request by tobySolutions**
## Description of the pull request

Found this other way to implement it that works too.

## Changes made

- Implement dynamic OG image generation
- Properly handle meta-data

However, since I am using the domain [akeru.ai](https://akeru.ai) for the base domain of the image here in the codebase, it does throw an error which I think would be fixed in production since I am using the production base URL

Screenshot 2024-04-18 at 15 35 25

## Checklist

- [ ] I have written tests
- [x] My code does not produce new errors
- [x] I gave myself a code review before asking others.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6189 @@
+{
```
Comment (GuiBibeau) [inline package-lock.json:1]: Let's delete that and potentially add package-lock.json to gitignore 😂

**Review by GuiBibeau — APPROVED** [review]
love vercel/og! Let's just fix the package.lock and good to merge

File: package-lock.json (hunk 1 above)
Comment (tobySolutions) [reply package-lock.json]: DONE GUI! 😁

File: packages/akeru-frontend/app/layout.tsx [hunk 2]
```diff
@@ -13,19 +13,39 @@ const montserrat = Montserrat({
...
+    },
   },
-
-  keywords: "Open source AI, Decentralized AI, AI, Assistant AI",
   twitter: {
+    title: "Akeru AI",
     card: "summary_large_image",
   },
-  metadataBase: new URL("https://akeru.ai"),
+  // verification: {
```
Comment (GuiBibeau) [inline packages/akeru-frontend/app/layout.tsx:45]: still need?

Comment (tobySolutions) [reply packages/akeru-frontend/app/layout.tsx:45]: Don't we need it?

Comment (GuiBibeau) [reply packages/akeru-frontend/app/layout.tsx:45]: I don't know to be honnest. If it's commented probably not?

Comment (tobySolutions) [reply packages/akeru-frontend/app/layout.tsx:45]: Ok, let me remove it for now.

Comment (tobySolutions) [reply packages/akeru-frontend/app/layout.tsx]: All done now.

**Pull request merged**